modelling/src/main/java/org/axonframework/modelling/saga/metamodel/SagaModel.java

```diff
@@ -32,30 +33,33 @@
 public interface SagaModel<T> {
 
     /**
-     * Returns the {@link AssociationValue} used to find sagas of type {@code T} that can handle the given {@code
-     * eventMessage}. If the saga type does not handle events of this type an empty Optional is returned.
+     * Returns the {@link AssociationValue} used to find sagas of type {@code T} that can handle the given
+     * {@code eventMessage}. If the saga type does not handle events of this type an empty Optional is returned.
      *
      * @param eventMessage The event to find the association value for
+     * @param context
```

Comment:
Missing some legacy JavaDoc here

Reply:
Javadoc has been added to this and relevant methods. Resolving.